doc/api/buffer.markdown

```diff
@@ -764,6 +764,13 @@ buffer.
     var b = new Buffer(50);
     b.fill("h");
 
+### buf.indexOf(value[, fromIndex])
```

Comment:
Possibly something like `byteOffset` would be better than `fromIndex`. The latter makes it sound like the character placement, but that's incorrect for multi-byte strings.